### PR TITLE
Add JSON data to the servlet and fetch and display on portfolio

### PR DESCRIPTION
**Pull request by dhruvi618**

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -23,6 +23,12 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
+			<groupId>com.google.code.gson</groupId>
+			<artifactId>gson</artifactId>
+			<version>2.8.6</version>
```
Comment (tcwanggoogle) [inline portfolio/pom.xml:30]: Indentation

Comment (dhruvi618) [reply portfolio/pom.xml:30]: I am seeing the correct indentation on Cloud Shell but for some reason it changes once pushed
Screenshot 2020-06-02 at 1 15 30 PM

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -19,14 +19,27 @@
...
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
     
+    private ArrayList<String> messages;
```
Comment (kmh287) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:29]: Does toJson specifically require an ArrayList or can we change the type of `messages` to List<String>?

The idea here is that you should use the most general type possible as that will give you flexibility later on if you want to change your code. For example if this were `private List<String> messages;` then you could change its type to a LinkedList just by changing the assignment on the first line of init.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -19,14 +19,27 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
```
Comment (kmh287) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:22]: Try to use granular imports whenever possible. So just java.util.List.

**Pull request merged**